### PR TITLE
[WIP] MTurk code removal

### LINKED ISSUE #2230: Get rid of MTurk residue
<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/2230/create?referer=github" target="_blank">Log Time</a>

### PR DESCRIPTION
**Pull request by fm3**
We decided it is time to remove the layers of indirection made necessary by mechanical turk tasks, since no one is using them anymore.

### TODO:
- [x] remove direct MTurk code
- [x] remove AssignmentConfig in projects
- [x] remove MTurk options in frontend
- [x] directly integrate OpenAssignments into Tasks
- [x] fix snapshot tests (=create new snapshots?)
- [x] create database evolutions for changed collections
- [x] remove schema files for removed collections

### Depends on:
- [done] edit task consistency (frontend changes in PR #2247 )
- [done] aggregated OpenAssignments (as enforced by Analysis Overview PR #2233 )

### Steps to test:
- Annotation / Task / Project management should still work, the MTurk-specific task creation form fields for are gone

### Issues:
- fixes #2230 

------
- [x] Ready for review

### CODE REVIEW

**Comment by fm3** [comment]
@jfrohnhofen how do you feel about old mturk projects in the database? removing the mturk code and assignmentConfiguration from the project case class kind of makes them unreadable. How should we handle them?

**Comment by fm3** [comment]
Is anyone still using `api/user/tasks/peek`? Our frontend seems not to. Do I need to ask in discuss? @normanrz

**Comment by normanrz** [comment]
@fm3 We should keep it: https://discuss.webknossos.org/t/show-upcoming-project-in-get-a-new-task-button-1624/52/5

**Comment by jfrohnhofen** [comment]
@fm3 I would like to remove them, but we should check back with @normanrz or @jstriebel to make sure, that they are properly backed up somewhere just in case...

File: app/controllers/TaskController.scala [hunk 1]
```diff
@@ -179,45 +179,24 @@ class TaskController @Inject() (val messagesApi: MessagesApi) extends Controller
...
         params.neededExperience,
         params.openInstances,
+        params.openInstances,
         _project = project.name,
         _script = params.scriptId,
         editPosition = params.editPosition,
         editRotation = params.editRotation,
-        boundingBox = params.boundingBox.flatMap { box => if (box.isEmpty) None else Some(box) })
+        boundingBox = params.boundingBox.flatMap { box => if (box.isEmpty) None else Some(box) },
+        priority = if(project.paused) -1 else project.priority)
```
Comment (jfrohnhofen) [inline app/controllers/TaskController.scala]: missing space

File: app/models/task/TaskAssignment.scala [hunk 2]
```diff
@@ -1,84 +1,13 @@
+/*
```
Comment (jfrohnhofen) [inline app/models/task/TaskAssignment.scala]: do we still need this file

**Pull request merged**